Solutions/Corvus.Extensions.Specs/TaskExtensionsFeatures/TaskExtensionsFeatureSteps.cs

```diff
@@ -37,24 +37,24 @@ public async void WhenICastTheTaskToAnInt()
             {
                 Task<int> result = sourceTask.CastWithConversion<int>();
                 this.context.Set(result, "Result");
-                await result;
+                await result.ConfigureAwait(false);
             }
             catch (InvalidCastException ex)
             {
-                await sourceTask;
+                await sourceTask.ConfigureAwait(false);
                 this.context.Set(ex, "Exception");
             }
         }
 
-        [Then(@"the task result should be an integer value (.*)")]
+        [Then("the task result should be an integer value (.*)")]
         public async Task ThenTheTaskResultShouldBeADoubleValue(int p0)
         {
             Task<int> task = this.context.Get<Task<int>>("Result");
-            int result = await task;
+            int result = await task.ConfigureAwait(false);
```

Comment:
You mustn't ConfigureAwait(false) in these specs. It messes with the async context.

Reply:
I've changed these all to be `ConfigureAwait(true)` in the specs.

Reply:
And now I've changed them back again as per discussion and #54